ratis-server-api/src/main/java/org/apache/ratis/server/DivisionInfo.java

```diff
@@ -20,6 +20,8 @@
 
 import org.apache.ratis.proto.RaftProtos.RaftPeerRole;
 import org.apache.ratis.proto.RaftProtos.RoleInfoProto;
+import org.apache.ratis.protocol.RaftPeer;
```

Comment:
Please remove this.  There is a checkstyle warning: https://github.com/apache/ratis/runs/6335329880?check_suite_focus=true#step:4:262 .

Reply:
Sorry because the Google CheckStyle I imported locally seems to be different from your style tool, so I did not format the code, so this line dependency is not removed